src/components/FormSections/Addresses.jsx

```diff
@@ -39,14 +40,24 @@ function Addresses(props){
     //Field requirements by state data
     const addressFieldsState = (nvrfStateFields.find(item => item.uuid === streetAddressField.uuid));
 
+    // Instructions for optional checkboxes (prev address, no address)
+    const addressCheckBoxInstructions = sanitizeDOM(noAddressField.instructions);
+    const addressCheckBoxesInstructions = sanitizeDOM(prevAddressField.instructions);
+
     return (
         <>
         <h2>{headings.step_label_2}</h2>
 
         {addressFieldsState && (
             <>
+            {!changeRegistrationVisible && (
+                <span className='usa-hint' id='addresses-checkbox_hint'>{addressCheckBoxInstructions}</span>
+            )}
             { changeRegistrationVisible && (
+                <>
+                <span className='usa-hint' id='addresses-checkbox_hint'>{addressCheckBoxesInstructions}</span>
```

Comment:
Minor formatting - change to addresses-checkbox-hint (no underscores) to be consistent